e2e/package.json

```diff
@@ -13,7 +13,7 @@
     "test-local:debug": "cross-env WORKBENCH_ENV=local PUPPETEER_HEADLESS=false DEBUG=true jest --maxWorkers=3",
     "test-staging": "cross-env WORKBENCH_ENV=staging jest --maxWorkers=3",
     "test-staging:debug": "cross-env WORKBENCH_ENV=staging PUPPETEER_HEADLESS=false DEBUG=true jest --maxWorkers=1",
-    "test:ci": "jest --ci --maxWorkers=3",
+    "test:ci": "jest --ci --maxWorkers=2",
```

Comment:
reduces concurrent threads to 2 to deal with flaky tests.

Reply:
I am ok with the change - is it well understood why more max workers causes an issue?

Reply:
Puppeteer troubleshooting tip: https://developers.google.com/web/tools/puppeteer/troubleshooting?hl=ja#running_puppeteer_on_circleci